packages/typography/src/index.ts

```diff
@@ -7,3 +7,6 @@ export {
   Disclaimer,
   Overline,
 } from './Typography';
+export { Link } from './Link';
+
+export declare const __TARGET__: 'web' | 'node';
```

Comment:
Why in here? It's only being used inside `Link`?

Reply:
Was going to use in spec, but then I didn't. Relocating!

Reply:
NVM. I had this in index.ts because I'm getting an error that `__TARGET__` is not defined when I declare it in a `.tsx` file, it only works for me when declared in a `.ts` file

Reply:
I wasn't having this issue earlier today 🤔

Reply:
![Screen Shot 2020-07-20 at 2 27 36 PM](https://user-images.githubusercontent.com/26016393/87972631-408ad680-ca95-11ea-8229-2916ef32d454.png)

Reply:
I'm seeing this issue coming from Menu too...